core/src/epicli/cli/helpers/build_saver.py

```diff
@@ -10,8 +10,11 @@
 MANIFEST_FILE_NAME = 'manifest.yml'
 SP_FILE_NAME = 'sp.yml'
 INVENTORY_FILE_NAME = 'inventory'
+INVENTORY_FILE_NAME_LEGACY = 'inventory/development'
```

Comment:
For legacy builds this path will not always be like that. Possible is: inventosy/staging, inventory/production, or other (was defined by user)

Reply:
But the it will be in the folder inventory/?

Reply:
This should be always in the folder inventory, but names can differ.

Reply:
Fixed